### PR TITLE
Issue2265 single zone controller fixes

### PR DESCRIPTION
**Pull request by dhblum**
This is for https://github.com/lbl-srg/modelica-buildings/issues/2265.  It:

- Changes package location and name of controller model
- Adds validation test
- Adds unit test reference results after implementing fix
- Updates documentation

### CODE REVIEW

**Comment by dhblum** [comment]
@mwetter I do not know why CI testing has not started on my last commit for this PR.

File: Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo [hunk 1]
```diff
@@ -1,5 +1,5 @@
-within Buildings.Air.Systems.SingleZone.VAV;
-model ChillerDXHeatingEconomizerController
+within Buildings.Air.Systems.SingleZone.VAV.BaseClasses;
```
Comment (mwetter) [inline Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo:1]: This rename of the model also needs to be added to the conversion script and the release notes.

File: Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo [hunk 2]
```diff
@@ -313,6 +316,11 @@ heating coil and a cooling coil.
 </html>", revisions="<html>
 <ul>
 <li>
+November 20, 2020, by David Blum:<br/>
```
Comment (mwetter) [inline Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo:319]: This is a backwards-compatible bug fix and needs to be added to `Buildings/package.mo` in the release notes.

**Review by mwetter — CHANGES_REQUESTED** [review]
@dhblum Please see inline comments for required changes to conversion script and additions to release notes in `Buildings/package.mo`.

**Comment by dhblum** [comment]
Ok thanks @mwetter.  I'll address the changes in the coming days.

File: Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo (hunk 2 above)
Comment (dhblum) [reply Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo:319]: I'm sorry, but I'm a little confused on how best to add to release notes given your comment here and also [this one](https://github.com/lbl-srg/modelica-buildings/pull/2266#discussion_r533476790).  This is a backwards-compatible bug fix, but the model is also moved and renamed.  I see in ``"new components have been added to existing libraries"`` models that have been moved and I see in ``"existing components have been improved in a non-backward compatible way"`` models that require conversion scripts.

Comment (dhblum) [reply Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo:319]: Added to release notes https://github.com/lbl-srg/modelica-buildings/pull/2266/commits/f7818c64c2bf782d4910ec5cf6c970a4ee679515.

File: Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo (hunk 1 above)
Comment (dhblum) [reply Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo:1]: Added to release notes in https://github.com/lbl-srg/modelica-buildings/pull/2266/commits/f7818c64c2bf782d4910ec5cf6c970a4ee679515 and conversion script in https://github.com/lbl-srg/modelica-buildings/pull/2266/commits/857a3995d9e24e2ec263c493561e5fbd8b0d0307.

**Comment by dhblum** [comment]
@mwetter I've addressed your comments in the latest two commits.  Let me know if those are correct and if you have any other comments.

**Comment by mwetter** [comment]
This is ready to merge when all tests pass.
@dhblum : Note that the conversion script is `Buildings/Resources/Scripts/Dymola/ConvertBuildings_from_7_to_8.0.0.mos`

**Pull request merged**